tests/kernel/install_ltp.pm

```diff
@@ -28,7 +29,7 @@ sub scc_we_enabled {
 }
 
 sub add_desktop_productivity_module {
-    if (check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
+    if (get_required_var('ARCH') eq 'x86_64' and check_var('DISTRI', 'sle') and sle_version_at_least('15')) {
```

Comment:
Do we actually want to use `and` instead of `&&`. I vaguely remember this operator has some strange precedence or behaviour.

Reply:
In this case it should not matter because we only use one of the two kinds. We shouldn't blindly mix both types though

Reply:
@richiejp Thx, I'll avoid this in the future.

Reply:
`&&` produces what we usually "expect" from "and", but I think that `and` does not do any wrong here.